test.js

```diff
@@ -393,8 +410,6 @@ if (!nodeVersion.startsWith('v16.')) {
 	});
 
 	test('can use TextDecoderStream', async t => {
-		// @todo Remove the following comment when dropping support for Node 16
-		// eslint-disable-next-line no-undef
```

Comment:
This linting error was being reported both locally and in CI, but suddenly stopped being reported just now. I am quite sure why (maybe a minor/patch release of a `devDependencies` used by `xo`?), but removing it now works.